.github/workflows/build-test-ship.yaml

```diff
@@ -76,11 +80,10 @@ jobs:
         run: |
           go vet ./...
 
-  # Build the docker images on PR or push to dev/master, ship on release
-  build-and-ship-image:
-    name: "Build ${{ matrix.service }} image"
+  # Build images for api-proctor and, on release, ship final images to ECR.
+  build-and-ship-images:
+    name: Build ${{ matrix.service }} images
     runs-on: ubuntu-latest
-    if: ${{ !github.event.pull_request.draft }}
```

Comment:
I removed this guard so that we can run API Proctor tests on draft PRs, which I think is worthwhile. We can tweak the job conditions if we find we're running API Proctor more often than is useful.